File: levelings/programming_logic/cap02/calculating_percentage/percentage.py
# - Read the value
value = input('Enter a value: R$ ')
value = float(value)

# - Read the percentage
p = input('Enter a percentage: ')
p = float(p)

# - Calculate the percentage
perc = value * p / 100

# - Calculate the addiction
addi = value + perc

# - Calculate the discount
dis = value - perc

# - Display
print('Percentage: ', perc)
print('Addition: ', addi)
print('Discount: ', dis)

File: levelings/programming_logic/cap02/calculating_percentage/percentage.java
import java.util.Scanner;

public class percentage {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        double value, p, perc, addi, disc;

        // - Read the value
        System.out.print("Enter the value: ");
        value = input.nextDouble();

        // - Read the percentage
        System.out.print("Enter the percentage: ");
        p = input.nextDouble();

        // - Calculate the percentage
        perc = value * p / 100;

        // - Calculate the addition
        addi = value + perc;

        // - Calculate the discount
        disc = value - perc;

        // - Display the percentage
        System.out.println("Percentage: " + perc);

        // - Display the addition
        System.out.println("Addition: " + addi);

        // - Display the discount
        System.out.println("Discount: " + disc);

        input.close();
    }
}
